src/System.Data.SqlClient/tests/ManualTests/SQL/UdtTest/UDTs/Address/Address.csproj

```diff
@@ -1,7 +1,11 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>$(NetFrameworkMinimum)</TargetFramework>
+    <TargetFramework>$(NetMinimum)</TargetFramework>
```

Comment:
Is this correct? Shouldn't we be running tests against .NET Framework?

Reply:
Answered in next comment